### DISCUSSION TITLE
GitHub student pack

### DISCUSSION THREAD

**Discussion by abdulwaarith0** [community/community#63970] [GitHub Education]

**Comment by DylanDevelops** [comment]
Does it currently have the old one locked in, or are you asking for future reference?

If you are currently trying to change it, email GitHub and tell them that you would like to request a school change in your account settings.

**Reply by abdulwaarith0** [reply]
It has the old one locked in

**Reply by DylanDevelops** [reply]
According to online, if you email them, they will change it after sending you some ways of verification. Use this official channel: [https://support.github.com/request?tags=docs-generic](https://support.github.com/request?tags=docs-generic)

Good luck! :)

**Reply by abdulwaarith0** [reply]
Thanks Dylan, but I still have an issue, how do I get their support email address, cause I was only able to create a ticket when I got to the support page, of which I have done twice now without getting any reply yet

**Reply by DylanDevelops** [reply]
I would give it until Monday/Tuesday. Most likely, they take off on the weekends plus if their support office is based in California, there is a hurricane slowly going up the coast which shut a bunch of offices down.
